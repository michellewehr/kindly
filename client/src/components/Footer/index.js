export default function Footer() {
  return (
    //todo needs Tailwind style
    <footer className="w-100 mt-auto bg-secondary p-4">
      <div className="container">&copy;2021 Kindly</div>
    </footer>
  );
}
