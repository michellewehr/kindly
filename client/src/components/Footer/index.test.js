import { render, cleanup, fireEvent } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Footer from './index';

afterEach(cleanup);

describe('Footer component renders', () => {

   it('renders', () => {
      render(<Footer />);
   });

   it('Matches snapshot DOM node structure', () => {
      const { asFragment } = render(<Footer />);
      expect(asFragment()).toMatchSnapshot();
   });
});