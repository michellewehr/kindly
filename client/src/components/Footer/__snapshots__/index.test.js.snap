// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders Matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <footer
    className="w-100 mt-auto bg-secondary p-4"
  >
    <div
      className="container"
    >
      Â©2021 Kindly
    </div>
  </footer>
</DocumentFragment>
`;
